raiden-ts/src/raiden.ts

```diff
@@ -1387,6 +1397,35 @@ export class Raiden {
     this.store.dispatch(withdraw.request(undefined, meta));
     return promise;
   }
+
+  /**
+   * Fetches an ordered list of suggested partners from provided, configured or first found PFS
+   *
+   * @param token - Token address to get partners for
+   * @param options - Request options
+   * @param options.pfs - PFS to use, instead of configured or automatic
+   * @returns Ordered array of suggested partners, with address and scoring values according to PFS
+   */
+  public async suggestPartners(
+    token: string,
+    options: { pfs?: RaidenPFS } = {},
+  ): Promise<SuggestedPartner[]> {
+    assert(Address.is(token), [ErrorCodes.DTA_INVALID_ADDRESS, { token }], this.log.info);
+    const tokenNetwork = this.state.tokens[token];
+    assert(tokenNetwork, ErrorCodes.RDN_UNKNOWN_TOKEN_NETWORK, this.log.info);
+    const pfs = decode(PFS, options.pfs ?? (await this.findPFS())[0]);
+
+    return await fromFetch(`${pfs.url}/api/v1/${tokenNetwork}/suggest_partner`, {
+      method: 'GET',
+      headers: { 'Content-Type': 'application/json' },
+    })
+      .pipe(
+        timeout(this.config.httpTimeout),
```

Comment:
Do we eventually do any retries here? I mean the method is asynchronous. It must not block the application with a correct design.

Reply:
No, the request here is one-shot, it may reject in case of failures, and it's up to the user to retry. You could offer a retry button in case of errors, but errors here shouldn't happen unless on network connectivity issues.